deployment/src/test/java/io/quarkiverse/wiremock/devservice/ConfigProviderResource.java

```diff
@@ -1,22 +1,33 @@
 package io.quarkiverse.wiremock.devservice;
 
+import static io.quarkiverse.wiremock.devservice.ConfigProviderResource.BASE_URL;
+
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.QueryParam;
 import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
 
 import org.eclipse.microprofile.config.ConfigProvider;
 
-@Path("/quarkus/wiremock")
+@Path(BASE_URL)
 @ApplicationScoped
 class ConfigProviderResource {
 
-    @Path("/devservices/config")
+    static final String BASE_URL = "/quarkus/wiremock/devservices";
+
+    @GET
+    @Path("/reload")
+    public Response reload() {
```

Comment:
Should be added to the docs too, right?

Reply:
Sorry, I don't get your point. It's just an endpoint for testing purposes. What's your intention to add this to the doc?